### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by S-BLH**
1. Created a repository by the name of palindrome-checker.
2. cloned it to my local machine.
3. changed the branch from main to development and then feature-branch.
4. created index.html, style.css and script.js files.
5. configured the linters.
6. added the necessary code to complete the project.

### CODE REVIEW

**Comment by latifa-wakili** [comment]
great you created good repo
using git workflow
no linters error
clean codes 
keep it up!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  <meta charset="UTF-8"></meta>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0″></meta>
+  <title>Palindrome Checker</title>
+  <style>
+    body {
+      font-family: Arial, sans-serif;
+      text-align: center;
+    }
+  </style>
+</head>
```
Comment (ahzamir) [inline index.html]: - Your code has issues in here solve them:
image

**Review by ahzamir — CHANGES_REQUESTED** [review]
@S-BLH 

- Well done you have implemented the most of project requirements but still your project is not complemented:

- Your project has no design and has issues in meta tags

**Comment by S-BLH** [comment]
> @S-BLH
> 
> * Well done you have implemented the most of project requirements but still your project is not complemented:
> * Your project has no design and has issues in meta tags

tnx sir now I implemented some designs and modified the errors, kindly review it back. @ahzamir

File: index.html [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+    button:hover {
+      background-color: darkred;
+    }
+
+    #result {
+      font-size: 24px;
+      margin-top: 20px;
+    }
+  </style>
```
Comment (ahzamir) [inline index.html]: - You have a separate file for your CSS code which is `style.css` move your styling code there.
  - Don't forget to link your CSS file in your HTML file using link tag in the head of your HTML file.

- Also your design is not responsive for mobile version please make your design responsive for the mobile version.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        const reversedInput = cleanedInput.split('').reverse().join('');
+        
+        if (cleanedInput === reversedInput) {
+          resultDiv.innerText = userInput + ' is a palindrome';
+        } else {
+          resultDiv.innerText = userInput + ' is not a palindrome';
+        }
+      }
+    });
+  </script>
```
Comment (ahzamir) [inline index.html]: - Same here you need to move your JS code to your `script.js` file and link your JS file using `<script>` tag at the end of your HTML body code.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,24 @@
...
+- JavaScript
+
+
+## Author 👩‍💻
+
+- LinkedIn: https://www.linkedin.com/in/sumita-balouch-959197309?utm_source=share&utm_campaign=share_via&utm_content=profile&utm_medium=ios_app
+- Email: sumitabalouch@gmail.com
+
+## License 📜
+MIT license.
```
Comment (ahzamir) [inline README.md]: - Add the Demo section with a screenshot and also add the Usage or How to use section and give users an instruction on how they can use your project.

**Review by ahzamir — CHANGES_REQUESTED** [review]
@S-BLH  

- Well done you solved the code issues but still your project needs some changes t get the final approval please check the final comments for more details:

**Comment by S-BLH** [comment]
@ahzamir as you said sir, I made it responsive for all mobile versions and separated the html, css and js from e/o, kindly review it.

**Review by ahzamir — APPROVED** [review]
### Hi @S-BLH 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- You did not deployed your branch please deploy your code.
- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
image

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

WinkThumbsUpGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Palindrome Checker project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a professional README.md file
✔️Your repo have been reviewed by your partner
✔️Your design is responsive

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Pull request merged**